src/index.html

```diff
@@ -112,5 +112,6 @@
 			 </ul>
 		 </div>
 	 </section>
+	 <script src="js/geolocation.js"></script>
```

Comment:
czemu to jest tutaj, a nie w jakims mainie?

Reply:
Przerzucę do maina, dodać atrybut async czy mam blokować parsowanie aż do uzyskania odpowiedzi użytkownika?